### PR TITLE
Moved Avro Schema verifier utility to helper:helper.

### PR DESCRIPTION
**Pull request by majisourav99**
Moved Avro Schema verifier utility to helper:helper.

### CODE REVIEW

File: helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/AvroSchemaVerifier.java [hunk 1]
```diff
@@ -25,12 +24,15 @@
...
   private final static AvroSchemaVerifier INSTANCE = new AvroSchemaVerifier();
+  private  static boolean isAvroVersionSupported = false;
 
   /**
    * Retrieves the verifier instance
    * @return
    */
   public static AvroSchemaVerifier get() {
+    AvroVersion avroVersion = AvroCompatibilityHelper.getRuntimeAvroVersion();
+    isAvroVersionSupported = avroVersion != AvroVersion.AVRO_1_10 && avroVersion != AvroVersion.AVRO_1_9;
```
Comment (radai-rosenblatt) [inline helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/AvroSchemaVerifier.java:35]: there are methods to compare avro version enums for laterThan etc. probably better here? (you want avro < 1.9)

Comment (radai-rosenblatt) [reply helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/AvroSchemaVerifier.java:35]: also, you want to make this call once, in static initialization ?

Comment (radai-rosenblatt) [reply helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/AvroSchemaVerifier.java:35]: and you want a comment explaining why and a TODO to fix this

Comment (majisourav99) [reply helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/AvroSchemaVerifier.java]: Hi Radai,
 I have addressed the comments, please take a look.

File: helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/AvroSchemaVerifier.java [hunk 2]
```diff
@@ -25,6 +24,13 @@
  */
 public class AvroSchemaVerifier {
   private final static AvroSchemaVerifier INSTANCE = new AvroSchemaVerifier();
+  private  static boolean isAvroVersionSupported = false;
```
Comment (radai-rosenblatt) [inline helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/AvroSchemaVerifier.java:27]: make final. also - double space

**Review by radai-rosenblatt — APPROVED** [review]

**Pull request merged**